File: src/main/java/com/neo/caption/ocr/property/AppInfoProperties.java
package com.neo.caption.ocr.property;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;

@Data
@ConfigurationProperties(prefix = "cocr.info")
public class AppInfoProperties {

    private String artifact;
    private String group;
    private String name;
    private String appLicense;
    private String javaVersion;
    private String springBootVersion;
    private String version;
    private String buildTimestamp;

}

File: src/main/kotlin/com/neo/caption/ocr/property/AppInfoProperties.kt
package com.neo.caption.ocr.property

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "cocr.info")
data class AppInfoProperties(
    val artifact: String,
    val group: String,
    val name: String,
    val appLicense: String,
    val javaVersion: String,
    val springBootVersion: String,
    val version: String,
    val buildTimestamp: String,
)